minio/api.py

```diff
@@ -541,7 +542,8 @@ def listen_bucket_notification(self, bucket_name, prefix='', suffix='',
 
     def fput_object(self, bucket_name, object_name, file_path,
                     content_type='application/octet-stream',
-                    metadata=None, sse=None, progress=None):
+                    metadata=None, sse=None, progress=None,
+                    part_size=MIN_PART_SIZE):
```

Comment:
I think it is better to add DEFAULT_PART_SIZE = MIN_PART_SIZE and use it instead of MIN_PART_SIZE, it is more clear and easy to change later.

Reply:
Done